docs: document set theory symbols (#5690)
NixOS/nix: doc/manual/src/glossary.md

    store. It can contain regular files, directories and symbolic
    links.  NARs are generated and unpacked using `nix-store --dump`
    and `nix-store --restore`.
  - `∅` \

>     The empty set symbol. In the context of package history, this denotes a package was not present prior to installation.

Reviewer: ```suggestion
    The empty set symbol. In the context of profile history, this denotes a package is not present in a particular version of the profile.
```
